gpg_group_chat/arguments.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='GPG Group Chat')
+    parser.add_argument('--client',
+                        default=False,
+                        action='store_true',
+                        help='Cliente mode')
+
+    parser.add_argument('--server',
+                        default=False,
+                        action='store_true',
+                        help='Server mode')
+
+    parser.add_argument('-t', '--target',
+                        metavar='<target>',
+                        default='127.0.0.1',
+                        help='target connect to the server')
```

Comment:
Maybe this message will be better: "an IP or a range of IP that is alowed to connect when running in server mode"